api/src/main/java/edu/cornell/mannlib/vitro/webapp/i18n/freemarker/I18nStringTemplateModel.java

```diff
@@ -83,13 +83,13 @@ public Object exec(List args) throws TemplateModelException {
 	 * and combines preProcessed string back to be used with online translations.
 	 * Substitutes arguments in message which is a part of preProcessed string  
 	 * @param preProcessed String "startSep + key + intSep + textString + intSep + message + endSep"
-	 * @param arguments that should be listed before message and substituted in the message itself 
+	 * @param args that should be listed before message and substituted in the message itself
 	 * @return
 	 */
 	private String getOnlineTranslationsFormattedMessage(String preProcessed, Object[] args) {
 		String[] parts = preProcessed.split(I18nBundle.INT_SEP);
 		final int messageIndex = parts.length -1;
-		String message = MessageFormat.format(parts[messageIndex], args);
+		String message = MessageFormat.format(parts[messageIndex].replaceAll("''", "'").replaceAll("'", "''"), args);
```

Comment:
@litvinovg please check do we need this here

Reply:
I think replace should be used instead of replaceAll as there is no regex used.
I suggest to extract method that does it instead of inline fixes.